force-app/main/default/aura/WorkItemKanbanView/WorkItemKanbanViewHelper.js

```diff
@@ -0,0 +1,95 @@
+({
+    setWorkItemList : function(component) {
+        var action = component.get("c.getWorkItems");
+        action.setCallback(this, function(response) {
+            var state = response.getState();
+            if (state === "SUCCESS") {
+                let workItemList = response.getReturnValue();
+                component.set("v.workItemList", workItemList);
+            }
+            else if (state === "INCOMPLETE") {
+                alert("INCOMPLETE OPERATION");
+                this.showToast("warning","Operation was not completed");
+            }
+                else if (state === "ERROR") {
+                    var errors = response.getError();
+                    if (errors) {
+                        if (errors[0] && errors[0].message) {
+                            alert("ERROR ON WORK ITEMS CALLBACK");
+                            console.log("Error message: " +  errors[0].message);
```

Comment:
remember to remove all the **console.log**s when uploading your code into the repository